index.html

```diff
@@ -1,42 +1,314 @@
-      <a class="nav-items" href="#"><i class="fas fa-bars"></i></a>
+<body>
+  <header>
+    <div class="jd-container">
+      <a class="nav-items p-color" href="#">John Doe</a>
+    </div>
+    <nav class="menu-icons">
+      <span>
+        <a class="nav-items p-color" href="#"><i class="fas fa-bars"></i></a>
+      </span>
     </nav>
+    <div class="collapse-menu">
+      <ul class="d-flex align-center inter-font">
+        <li>Portfolio</li>
+        <li>About</li>
+        <li>Contact</li>
+        <li><i class="far fa-envelope"></i></li>
+      </ul>
+    </div>
   </header>
 
-  <section class="main">
-    <h1 class="Welcome">Hey There. <br> I’m Alex</h1>
-    <h2 class="Welcome-h2">I’am a Software Developer</h2>
-    <p class="content">I can help you build a product , feature <br> 
-      or website Look through some of my <br> 
-      work and experience! If you like what you see <br> 
-      and have a project you need coded, <br> 
-      don’t hestiate to contact me.</p>
-    
-    <ul class="social-media">
-      <li class="media-link"><a href="#"><img src="imgs/github.svg" alt=""></a></li>
-      <li class="media-link"><a href="#"><img src="imgs/Linkedin-icon.svg" alt=""></a></li>
-      <li class="media-link"><a href="#"><img src="imgs/angel-list.svg" alt=""></a></li>
-      <li class="media-link"><a href="#"><img src="imgs/twitter.svg" alt=""></a></li>
-      <li class="media-link"><a href="#"><img src="imgs/medium.svg" alt=""></a></li>
+  <main>
+    <section class="headline">
+      <div class="content-container">
+        <h1 class="primary-text-h1 fonts-1">Hey There. I’m Alex</h1>
+        <h2 class="primary-text-h2 fonts-2">I’m a Software Developer</h2>
+        <p class="content fonts-3 p-color">I can help you build a product, feature or website Look through some of my
+          work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+      </div>
+      <ul class="social-media">
+        <li class="media-link">
+          <a href="#"><img src="imgs/github.svg" alt="Chima's Github">
+          </a>
+        </li>
+        <li class="media-link">
+          <a href="#"><img src="imgs/Linkedin-icon.svg" alt="Chima's LinkedIn">
+          </a>
+        </li>
+        <li class="media-link">
+          <a href="#"><img src="imgs/angel-list.svg" alt="Chima's Angelist">
+          </a>
+        </li>
+        <li class="media-link">
+          <a href="#"><img src="imgs/twitter.svg" alt="Chima's Twitter">
+          </a>
+        </li>
+        <li class="media-link">
+          <a href="#"><img src="imgs/medium.svg" alt="Chima's Blog">
+          </a>
+        </li>
+      </ul>
+    </section>
+
+    <section class="recent-works">
+      <div class="h2-and-line">
+        <h2 class="title fonts-1 fw-n">My Recent Works</h2>
+      </div>
+      <section class="frame44">
+        <ul class="card">
+          <li class="image-box">
```

Comment:
![image](https://user-images.githubusercontent.com/43698511/130325875-e2ae0c92-6852-48a1-9fff-2ddf5cc0db17.png)

Your project looks a bit different from the original design.
- [ ]  As you can see in the picture the my works section doesn't take all the page width it's centered and has some margin right and left. 
- [ ] The featured project image takes 60% of the page's width and is aligned to the left, the description takes the remaining 40% and is aligned to the right.
- [ ] For the other projects. The description of each of these projects is on top of their image with a dark opacity with white color text. When hovered on with the mouse the description and dark opacity disappear and a See Project button appears. See the picture bellow
![image](https://user-images.githubusercontent.com/43698511/130326067-2ed66c4b-743d-49c0-9490-8ccde34cd0bb.png)
Kindly fix these issues. Following the original design is required for this project. As it's an important skill for a front-end dev